src/App.js

```diff
@@ -147,10 +166,16 @@ function App() {
   );
 
   const message = useMemo(() => {
-    if (!guesses.length) return "Guess the five letter word";
-    if (guesses.length > 0 && !done) return "You're making progress!";
-    if (done && giveup === false) return `Congratulations! It took you ${guesses.length} guesses.`;
-    if (done && giveup === true) return `You gave up after ${guesses.length-1} guesses.`;
+    const r = Math.floor(Math.random() * 2);
```

Comment:
I think we should allow each array to have a different number of elements. And thus, we should pick a random number based on the array chosen. Perhaps we should add a function like:
```
const randomMessage = (messages) => messages[Math.floor(Math.random() * message.length)];
```

Reply:
OK. I got this implemented too. "message" vs "messages" in your suggested code was giving me issues.